client/app/projects/blog/blog.controller.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+angular.module('observatory3App')
+.controller('ProjectsBlogCtrl', function ($scope, $http, $stateParams) {
+    $scope.load = function() {
```

Comment:
Is this something that needs to be run like this? Not sure the function is needed.